src/main/kotlin/cacophony/Constants.kt

```diff
@@ -0,0 +1,4 @@
+package cacophony
```

Comment:
Good idea to actually create this file, we may later fix other places to get comment_start from there.

Reply:
imo not in this pr though, looking at our code i think some refactoring is inevitable...